.github/workflows/master_ci.yml

```diff
@@ -216,7 +216,7 @@ jobs:
           mkdir build
           cd build
           export CMAKE_GENERATOR="MSYS Makefiles"
-          /mingw64/bin/cmake .. -DCMAKE_BUILD_TYPE=RELEASE -DKAHYPAR_DOWNLOAD_BOOST=On -DKAHYPAR_CI_BUILD=ON -DKAHYPAR_ENABLE_HIGHEST_QUALITY_FEATURES=OFF -DKAHYPAR_ENABLE_LARGE_K_PARTITIONING_FEATURES=OFF
```

Comment:
Why is `-DKAHYPAR_DOWNLOAD_BOOST=On` removed here?

Reply:
Because the Boost MSYS2 package is already downloaded prior and therefore there is no need to build it again.